Tests/FirebaseTests/Tests/DynamicLinks/DynamicLinksRoutesTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import Firebase
+@testable import Vapor
+
+final class DynamicLinksRoutesTests: XCTestCase {
+
+    func test_routeRequests_should_beInJson_when_encodedBody() throws {
+        // Given
+        let request = DynamicLinkRequest(dynamicLink: DynamicLinkInfo(link: "http://link"),
+                suffix: Suffix(option: .SHORT))
+        let expectedJson = "{\"suffix\":{\"option\":\"SHORT\"},\"dynamicLinkInfo\":{\"link\":\"http:\\/\\/link\"}}"
```

Comment:
not a very solid way to compare a JSON. What if the encoder add some newlines by maybe setting a prettifier option?
A better test could be to try to decode it using a JSONDecoder and check it doesn't fails? 🤔